src/api/json/catalog.json

```diff
@@ -1623,6 +1623,14 @@
       ],
       "url": "https://json.schemastore.org/ui5-manifest"
     },
+    {
+      "name": "manifest.json",
```

Comment:
Sorry, one last thing: this name worries me a bit - others use more speaking names for such generic manifest.json, e.g.: https://github.com/silkearians/schemastore/blob/ac0e058c8677be0a00127586c02e379afce85487/src/api/json/catalog.json#L1744

Should we make it definitely unique by calling it **UI5 Manifest** 
and align the description to: **UI5 Manifest (manifest.json)**

But I would be also fine with the approach right now if you don't fear this generic name...

Reply:
done (also updated the commit message above